src/pages/DeleteTour.js

```diff
@@ -1,55 +1,55 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Container from '../components/Container';
+import { fetchTours, deleteTour } from '../redux/reducers/tours';
 
 const DeleteTour = () => {
-  const tours = [
-    {
-      id: 1,
-      title: 'Tour 1',
-    },
-    {
-      id: 2,
-      title: 'Tour 1',
-    },
-    {
-      id: 3,
-      title: 'Tour 1',
-    },
-    {
-      id: 4,
-      title: 'Tour 1',
-    },
-    {
-      id: 5,
-      title: 'Tour 1',
-    },
-  ];
+  const token = useSelector((state) => state.token.userData.token);
+  const tours = useSelector((state) => state.tours.package);
+  const dispatch = useDispatch();
 
-  const deleteTour = (e) => {
+  useEffect(() => {
+    dispatch(fetchTours());
+  }, []);
+
+  const handleDelete = (e) => {
     e.preventDefault();
+    const id = Number(e.target.getAttribute('data-id'));
+    dispatch(deleteTour({ token, id }));
   };
 
   return (
     <Container>
       <h1 className="text-center pt-12 text-2xl font-bold">Delete a Tour</h1>
-      <div className="flex flex-col py-2 px-4">
-        {tours.map((tour) => (
+      {tours
+        && tours.map((tour, index) => (
           <form
             key={tour.id}
-            onSubmit={deleteTour}
-            className="grid grid-cols-12 p-3 mb-2 border border-gray rounded items-center"
+            onSubmit={handleDelete}
+            data-id={tour.id}
+            className="flex flex-col py-2 px-4"
           >
-            <span className="col-span-2">{tour.id}</span>
-            <p className="col-span-8">{tour.title}</p>
-            <button
-              type="submit"
-              className="p-2 bg-red text-white text-sm rounded col-span-2 md:col-span-1"
-            >
-              Delete
-            </button>
+            <div className="grid grid-cols-12 p-3 mb-2 border border-gray rounded items-center">
+              <span className="col-span-2">{index + 1}</span>
+              <p className="col-span-8">{tour.name}</p>
+              {tour.available && (
+                <button
+                  type="submit"
+                  className="p-2 bg-red text-white text-sm rounded col-span-2 md:col-span-1"
+                >
+                  Delete
```

Comment:
Verify with the backend team that only the owner or Admin of the Tour can delete it. Also on the delete page, only tours created by the logged user should be shown.